Dockerfile

```diff
@@ -9,7 +9,7 @@ COPY --chown=carma . /home/carma/autoware.auto
 RUN chmod -R 775 /home/carma/autoware.auto/docker/checkout.bash
 RUN chmod -R 775 /home/carma/autoware.auto/docker/install.sh
```

Comment:
Nit: Let's move this down just before `RUN /home/carma/autoware.auto/docker/install.sh`
Otherwise, everytime we change something in install.sh, it is redoing the checkout.bash and not caching.